_sass/hydeout/_layout.scss

```diff
@@ -270,7 +279,8 @@ body {
 }
 
 #sidebar nav > .icon {
-  font-size: 1.5rem;
+  color: $sidebar-link-color;
+  font-size: 1.35rem;
```

Comment:
Should be `1.2rem` and `font-weight: 500` to match the other links

Reply:
I'm not sure, the icons would be smaller in mobile if it's `1.2rem`

Reply:
`font-weight`applied.

Reply:
anyway I made svg icon appeared bigger so font size is resolved